source/2020-03-13-the-ember-times-issue-139.md

```diff
@@ -30,13 +30,11 @@ You can [watch all six talks today on YouTube](https://www.youtube.com/playlist?
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Ember-models-table released version 3.0.0 🌟](https://twitter.com/oonechiporenko/status/1235571293530902531)
```

Comment:
Because the word Ember appears as a part of an addon's name, I think we could make the word lowercase.
```suggestion
## [ember-models-table released version 3.0.0 🌟](https://twitter.com/oonechiporenko/status/1235571293530902531)
```